edgexfoundry/device-sdk-go
# fix: [WIP BUG] Fix hang on subscribe errors during bootstrap

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Run non-secure EdgeX stack
Build device simple using this branch
run device simple with just `-cp` flag
Verify the service doesn't hang and exits with errors
run device simple with just `-cp -d` flags DEBUG logging env override
Verify service bootstraps successfully and starts publishing events 

## New Dependency Instructions (If applicable)
